recipe/meta.yaml

```diff
@@ -24,26 +24,30 @@ requirements:
     - setuptools
     - blas 1.1 {{ variant }}
     - openblas 0.2.20|0.2.20.*
-    - gsl
-    - fftw
+    - gsl     # [not win]
+    - fftw    # [not win]
     - glpk
-    - suitesparse 4.5.4
+    - suitesparse 4.5.4   # [not win]
+    - suitesparse 5.1.2   # [win]
```

Comment:
Using 4.5.\* should be fine. It's been stable across minor versions. Same story with 5.1.\*. 5.1.0 added new functionality, but didn't break anything from 5.0.\*.

ref: https://abi-laboratory.pro/tracker/timeline/suitesparse/

Reply:
I think using suitesparse 4.5.*  may still be problematic on macOS if there is ever a suitesparse 4.5.5 (see conda-forge/suitesparse-feedstock#22 and conda-forge/suitesparse-feedstock#23). I remember there was an issue with cvxopt at some point when you upgraded to from suitesparse 4.5.3 to 4.5.4.

Reply:
Right, sorry, I misread that issue. Pinning tightly is fine for now.